src/read_cache_fs_extension.cpp

```diff
@@ -67,6 +67,12 @@ static void LoadInternal(DatabaseInstance &instance) {
     }
   }
 
+  auto &config = DBConfig::GetConfig(instance);
+  config.AddExtensionOption(
+      "cached_http_cache",
```

Comment:
config should be renamed, at least it should contain words like `location` or `directory`